File: FA-backend/src/FitnessApp.InterfaceLayer/Dtos/Classes/User/CreateUserDto.cs
using FitnessApp.InterfaceLayer.Dtos.Enums;

namespace FitnessApp.InterfaceLayer.Dtos.Classes.User;

public class CreateUserDto
{
    public UserSexEnum _sex { get; set; }
    public EduAttainmentEnum _edu { get; set; }
    public int _birthYear { get; set; }
    
    public bool? _athlete { get; set; }
    public bool? _smoker { get; set; }
}

File: FA-frontend/src/app/create-user-dto.ts
enum Sex{
    undefined,
    male,
    female,
}

enum Education{
    undefined,
    ZS,
    SS,
    VS,
}

export class CreateUserDto {
    public _sex: Sex;
    public _education: Education;
    public _birthYear: number;
    public _athlete: boolean;
    public _smoker: boolean;

    constructor() {
        this._sex = Sex.undefined;
        this._education = Education.undefined;
        this._birthYear = -1;
        this._athlete = false;
        this._smoker = false;
    }
}
